test/unit/startup-tasks/task-start-consumers-test.js

```diff
@@ -0,0 +1,59 @@
+require('require-self-ref')
+
+const test = require('ava')
+
+const sinon = require('sinon')
+const proxyquire = require('proxyquire')
+proxyquire.noPreserveCache()
+
+test.beforeEach('inject mocks', async (t) => {
+  const sandbox = sinon.sandbox.create()
```

Comment:
Super minor - but this is creating unnecessary extra objects. You could just create the sandbox once and then `.restore()` it in `afterEach` (which you're already doing).

Reply:
Because of the way that `ava` runs tests in parallel, it's not safe to create a single `sandbox` for each test to use.